modules/nw-udn-additional-config-details.adoc

```diff
@@ -37,8 +40,27 @@ When enabled, OVN-Kubernetes applies the IP configuration to the SDN infrastruct
 **Disabled:** +
 When disabled, OVN-Kubernetes only assigns MAC addresses and provides layer 2 communication, which allows users to configure IP addresses. `Disabled` is only available for layer 2 (secondary) networks. By disabling IPAM, features that rely on selecting pods by IP, for example, network policy, services, and so on, no longer function. Additionally, IP port security is also disabled for interfaces attached to this network. The `subnets` field must be empty when `spec.ipam.mode` is set to `Disabled.`
 
-|`spec.layer2.mtu` and `spec.layer3.mtu`
+|`spec.network.<topology>.mtu`
+|`spec.<topology>.mtu`
 |integer
 |The maximum transmission units (MTU). The default value is `1400`. The boundary for IPv4 is `576`, and for IPv6 it is `1280`.
 
-|====
\ No newline at end of file
+|`spec.network.localnet.vlan`
+|N/A
+|object
+|This field is optional and configures the virtual local area network (VLAN) tagging and allows you to segment the physical network into multiple independent broadcast domains.
+
+|`spec.network.localnet.vlan.mode`
+|N/A
+|object
+|Acceptable values are `Access`. A value of `Access` specifies that the network interface belongs to a single VLAN and all traffic will be labelled with an `id` that is configured in the `spec.network.localnet.vlan.mode.access.id` field. The `id` specifies the VLAN `id` (VID) for access ports. Values must be an integer between 1 and 4094.
+
+|`spec.network.localnet.physicalNetworkName`
+|N/A
+|string
+|Specifies the name for a physical network interface. The value you specify must match the `network-name` parameter that you provided in your Open vSwitch (OVS) bridge mapping.
```

Comment:
@ormergi add your request here.

Reply:
Would it make sense a link to the doc where we describe how to configure the OVS bridge mapping?